system/genext2fs/genext2fs.info

```diff
@@ -1,8 +1,8 @@
 PRGNAM="genext2fs"
-VERSION="1.4.1"
+VERSION="1.5.0"
 HOMEPAGE="http://genext2fs.sourceforge.net"
-DOWNLOAD="http://downloads.sourceforge.net/project/genext2fs/genext2fs/1.4.1/genext2fs-1.4.1.tar.gz"
-MD5SUM="b7b6361bcce2cedff1ae437fadafe53b"
+DOWNLOAD="https://github.com/bestouff/genext2fs/archive/refs/tags/v1.5.0.tar.gz"
+MD5SUM="8730adbb3145c8d62c15b9ef254429db"
```

Comment:
Please use the guideline to get consistent output
https://slackbuilds.org/GITHUB_URLs.txt

Reply:
fixed, thx